### PR TITLE
Make max batch size optional

### PR DESCRIPTION
**Pull request by wangshan**
Related to PR https://github.com/ietf-wg-ppm/draft-ietf-ppm-dap/pull/530 in core DAP

### CODE REVIEW

File: draft-wang-ppm-dap-taskprov.md [hunk 1]
```diff
@@ -192,6 +192,10 @@ struct {
 } QueryConfig;
 ~~~
 
+In particular, if `query_type` is `fixed_size` and `max_batch_size` is 0,
```
Comment (cjpatton) [inline draft-wang-ppm-dap-taskprov.md:195]: Grammar: "In particular" is used to refine a previous statement, whereas here it's beginning a statement. I think what you want to do instead is start the paragraph off with "The max batch size for the `fixed_size` query is optional."

File: draft-wang-ppm-dap-taskprov.md [hunk 2]
```diff
@@ -192,6 +192,10 @@ struct {
...
+then it means `max_batch_size` is optional. Aggregators should provision the
+task without a `max_batch_size` limitation.
```
Comment (cjpatton) [inline draft-wang-ppm-dap-taskprov.md:197]: What does this sentence mean?

**Review by cjpatton — COMMENTED** [review]
Setting `max_batch_size` to `0` seems reasonable. Let's hold off on merging until the PR against the core DAP spec gets merged.

**Comment by wangshan** [comment]
Made the text more clear, @cjpatton pls have another pass, the DAP PR is merged, we can merge this once it's approved

**Pull request merged**